<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputForm</class>
 <widget class="QDialog" name="InputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запрос информации</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Введите данные&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SurnameInput">
       <property name="inputMask">
        <string>&gt;A!&lt;aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="placeholderText">
        <string>Введите фамилию</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameInput">
       <property name="inputMask">
        <string>&gt;A!&lt;aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="placeholderText">
        <string>Введите имя</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PatronymicInput">
       <property name="inputMask">
        <string>&gt;A!&lt;aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="placeholderText">
        <string>Введите отчество</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PhoneNumberInput">
       <property name="inputMask">
        <string>9-(999)-999-99-99</string>
       </property>
       <property name="text">
        <string>-()---</string>
       </property>
       <property name="maxLength">
        <number>17</number>
       </property>
       <property name="placeholderText">
        <string>Введите номер телефона</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Адрес</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AddressInput">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>Введите адрес</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="SubmitButton">
     <property name="text">
      <string>Принять</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
